src/System.Management.Automation/engine/CommandCompletion/CompletionHelpers.cs

```diff
@@ -55,36 +55,68 @@ internal static IEnumerable<CompletionResult> GetMatchingResults(
             }
         }
 
+        /// <summary>
+        /// Handles and processes a string that starts and/or ends with single or double quotes.
+        /// Removes the front quote if present, and optionally the back quote if it matches the front quote.
+        /// Updates the input string by reference and returns the type of quote used.
+        /// </summary>
+        /// <param name="wordToComplete">
+        /// The string to process, which may be surrounded by single or double quotes.
+        /// This parameter is passed by reference and is updated to exclude processed quotes.
+        /// </param>
+        /// <returns>
+        /// Returns a string representing the quote type used (' or ") if a front quote is detected.
```

Comment:
Please don't use special characters.
```suggestion
        /// Returns a string representing the quote type used (single or double) if a front quote is detected.
```